.github/workflows/sphinx-pages.yaml

```diff
@@ -0,0 +1,36 @@
+name: Update Sphinx Docs
+on: [push]
```

Comment:
I'm not positive, but I think this is what created by github actions inf loop.
I had to change it to:
```
on:
  push:
    branches:
      - master
```

Reply:
Just added this change